doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.6.99-cs2_to_2_7_environment.rst

```diff
@@ -0,0 +1,255 @@
+2_6_99_cs2:orphan:
+
+.. _migration_cs3_to_2_7_env:
+
+Update your development environment for |NCS| v2.7 (for v2.6.99-cs2 users)
+##########################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes how to update your development environment for the nRF54H20 DK from |NCS| v2.6.99-cs2 to |NCS| v2.7.
+
+The main development environment changes introduced by 2.7 for the nRF54H20 DK are the following:
+
+..
+   ##TODO
```

Comment:
and also than VS Code is not supported according to my best knowledge. Please confirm this

Reply:
CC @shanthanordic

Reply:
@shanthanordic Hi, could you please confirm this?